Model/Sales/Repository/PackageRepository.php

```diff
@@ -104,6 +111,22 @@ public function fitInDigitalStamp()
         return true;
     }
 
+    /**
+     * @param \Magento\Quote\Model\Quote\Item[] $products
+     */
+    public function disableCheckoutWithProduct($products)
+    {
+        foreach ($products as $product) {
+            $getDisabledOption = (int) $this->getAttributesFitInOptions($product, 'disable_checkout');
```

Comment:
method naam is niet duidelijk. Liever globale benaming aanmaken

Reply:
Done